### DISCUSSION TITLE
Lowest temperature setting for TS80 or TS100 running IronOS

### DISCUSSION THREAD

**Discussion by beyeriii** [Ralim/IronOS#1012] [Q&A]
The stock TS80 and TS100 temperature range is 100ºC to 400ºC

I am looking for something that will produce temperature around 42ºC. 

Does a TS80 or TS100 running IronOS allow adjustment to ~42ºC?

Or is there a different soldering iron which can go down to ~42ºC?

**Comment by Ralim** [comment]
So your main issue will be that the PID for temperature control is not particularly stable at low temperatures with a high input voltage.

What is your use case?

If you want this to work I would suggest using a ts100/Pinecil at low inputs (12V) or a TS80P at 5/9V.
Reducing the input voltage will reduce max power a lot and will smooth out the PID.
But the firmware _should_ let you do this if you wanted.
You would probably want to change the small increments down to 1 to let you fine tune it in.
